301-create-ad-forest-with-subdomain/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "adAdministrator"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "VMSize": {

```

Comment:
remove the params that have default values

Reply:
done.